README.md

```diff
@@ -24,7 +24,7 @@ Download the [minified](https://raw.githubusercontent.com/ml5js/ml5-library/mast
 ```
 or 
 ```javascript
-<script src="https://unpkg.com/ml5@0.1.1/dist/ml5.min.js" type="text/javascript"></script>
+<script src="https://unpkg.com/ml5/dist/ml5.min.js" type="text/javascript"></script>
```

Comment:
```suggestion
<script src="https://unpkg.com/ml5@0.1.3/dist/ml5.min.js" type="text/javascript"></script>
```

Due to the rapidly changing nature of the project it's helpful when the specific version is referenced. A bit of a pain perhaps, but best to just keep this up to date with the latest?

Reply:
I understand, but I also found that using the tag @latest would do exactly this, using the latest version, would that be better?

Reply:
What about including two options for including ml5.js with unpkg.com on the README? 

1. This is how you download the latest version.
2. This is how you point to a specific version.

Reply:
Yeah, that might be a good idea. @montoyamoraga, feel free to submit an update to reflect those changes